src/web_app_skeleton/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.8'
+services:
+  lucky:
+    build:
+      context: .
+      dockerfile: docker/development.dockerfile
+    environment:
+      DATABASE_URL: postgres://postgres@postgres:5432/postgres
+    volumes:
+      - type: bind
+        source: .
+        target: /app
+      - type: volume
```

Comment:
it's not clear to me why these have to be separate modules? Kind of ties into what I mentioned about build steps typically being performed at image build time I guess

Reply:
Since compose is overlaying the app source code at runtime onto the built container image, these need to be volumes. If the system architecture (or operating system) differs from the container, node packages which have a compiled component will fail with weird and difficult to comprehend error messages when they're imported into the container.